src/inset_state/inset_state.cpp

```diff
@@ -895,10 +895,11 @@ double InsetState::target_area_at(const std::string &id) const
   }
 }
 
-double InsetState::total_inset_area() const
+double InsetState::total_inset_area(bool original_area) const
 {
+  auto &geo_divs = original_area ? geo_divs_original_transformed_ : geo_divs_;
```

Comment:
Storing the area in a variable would require us to update that variable each time we update the `geo_divs`, which would require extra book keeping in many places, and this will increase the possibilities of bugs.